### ISSUE TITLE
Disable by post types

### ISSUE THREAD

**Issue by dshanske** [pfefferle/wordpress-webmention#243]
Support not displaying or storing webmentions of specific types, such as likes.

See https://chat.indieweb.org/2019-12-23/1577119704084100

**Comment by tw2113** [comment]
I'm assuming until told otherwise that types here means the items from Post Kinds.

If there's a function we could check for definition of, that returns either default or enabled post kinds, then that could be useful for creating a setting for this.

**Comment by dshanske** [comment]
@tw2113 Not exactly. The request was to discard webmentions of specific types...so this would be after post type discovery... currently this is done in Semantic Linkbacks but will be moving here

**Comment by edent** [comment]
Anything I can do to help with this?  A [tweet](https://twitter.com/edent/status/1263430540260257799) has sent me a couple of hundred webmentions. I enjoy having the comments replicated on my site, but I don't need the likes and reposts.

It's quite tedious to click delete on each one. 

I foresee an interface like:

**Which Webmentions do you want to receive?**
* [ ]  Likes
* [x]  Replies
* [ ]  Reposts

**Comment by dshanske** [comment]
I'll see if we can do something interim for you.
